CMakeLists.txt

```diff
@@ -1,12 +1,17 @@
 cmake_minimum_required(VERSION 3.1)
 project(gpio++ VERSION 4.1)
 
+include(GNUInstallDirs)
+
+find_library(asio REQUIERED FATAL)
```

Comment:
You have a typo in `REQUIRED`.

Reply:
Also in order for `find_library` to work, you need `FindAsio.cmake` module, which needs to be written...

Maybe for now just add a dependency in your ebuild and I will try to work it some time later.